index.js

```diff
@@ -0,0 +1,58 @@
+const booksContainer = document.querySelector('.books');
+const titleInput = document.getElementById('title-input');
+const authorInput = document.getElementById('author-input');
+
+const addBtn = document.querySelector('.add-btn');
+
+const Books = JSON.parse(localStorage.getItem('books')) || [];
+
+function addBook(title, author, id) {
+  const book = {
+    title,
+    author,
+    ID: id,
+  };
+  Books.push(book);
+  localStorage.setItem('books', JSON.stringify(Books));
+  // eslint-disable-next-line no-use-before-define
+  displayBooks();
+}
+
+function removeBook(index) {
+  Books.splice(index, 1);
+  localStorage.setItem('books', JSON.stringify(Books));
```

Comment:
- [ OPTIONAL] I can see you used localStorage to store the `book` collection, which is a good approach for preserving data :+1: However, it does not handle the case where there is no data in `localStorage`, which could cause issues when loading the application for the first time. It is recommended to check if there is data in localStorage and initialize the `book` collection array accordingly. :wink: 

I think this how it should be :thinking: 
```
let books = [];

if (localStorage.getItem("books")) {
  books = JSON.parse(localStorage.getItem("books"));
} else {
  books = [];
}
```

_note: That will check if there is any data stored in the `books` key in `localStorage.` If there is, it parses the data and initializes the `books` array with it. If there is no data, it creates an empty array. You can place this code at the beginning of your project to ensure that the book collection array is initialized properly. :clinking_glasses:_

Reply:
@Alfredbis29 Thanks for saying that, 
I found a more convenient way of writing your proposed code which does the exact same thing as you intend,
`const books = JSON.parse(localStorage.getItem('books')) || [];`
using the OR operator assigns an empty array if Local storage is empty.
![Screenshot (96)](https://user-images.githubusercontent.com/81446846/234418132-f142b19e-8de2-47e8-b42e-6bbdd373ce62.png)
As you can see it stored an empty array